gtest_parallel_tests.py

```diff
@@ -143,6 +159,35 @@ def test_run_task_basic(self):
 
     self.assertEqual(task_manager.global_exit_code, 1)
 
+  @unittest.skipIf(sys.platform == 'win32', 'non-Windows specific')
```

Comment:
The test will fail. save_file will end with .cache/gtest-parallel while test expects AppData/Local/gtest-parallel.

I don't see a point in running such test. Can you please explain why you need it because otherwise I don't understand how I can fix it? As for me having to 2 tests, 1 for Windows and 1 for anything else, should be enough.